vllm/v1/spec_decode/eagle.py

```diff
@@ -298,8 +303,10 @@ def load_model(self, target_model: nn.Module) -> None:
         draft_attn_layer_names = (
             get_layers_from_vllm_config(self.vllm_config, Attention).keys() -
             target_attn_layer_names)
-        assert len(draft_attn_layer_names) == 1
-        self.attn_layer_name = next(iter(draft_attn_layer_names))
+
+        # AttentionMetadata will be shared across all draft layers
+        # if draft model has > 1 layers
```

Comment:
Why do you need such a comment here? Can we remove it?

Reply:
This was to replace original assumption that there is only one layer name for eagle, removed